fhir2-appnt-api/pom.xml

```diff
@@ -0,0 +1,93 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>appointments</artifactId>
+        <groupId>org.openmrs.module</groupId>
+        <version>1.2-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>org.openmrs.module</groupId>
+    <artifactId>fhir2-appointment-api</artifactId>
+    <version>${openmrsModuleFhirVersion}</version>
```

Comment:
Shouldn't the version here inherit from the parent module, i.e., be tied to the Bahmni appointment module version rather than the FHIR module version?